### PR TITLE
Fix Security alert

### PR DESCRIPTION
**Pull request by sanderegg**
## What do these changes do?

- [x] set Jinja2>=2.10.1 to prevent security issue
- [x] fixed make clean recipe
- [x] auto install pip-tools when creating requirements

## Checklist

- [x] I think the code is well written
- [x] Unit tests for the changes exist
- [x] Documentation reflects the changes
- [x] **Runs in the swarm**
- [x] If you design a new module, add your user to .github/CODEOWNERS

[waffle.io]:https://waffle.io/marketing-assets/documents/waffleio_cheatsheet_v1.pdf?utm_source=blog&utm_medium=cheatsheet-ctabutton&utm_campaign=cheatsheet

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -17,7 +17,7 @@ future==0.17.1            # via cookiecutter
 idna==2.8                 # via requests
 isort==4.3.10             # via pylint
 jinja2-time==0.2.0        # via cookiecutter
-jinja2==2.10              # via cookiecutter, jinja2-time
+jinja2==2.10.1
```
Comment (pcrespov) [inline requirements.txt]: You should NOT modify ``requirements.txt`` but ``requirements.in`` and rerun pip-tools (e.g. using https://github.com/ITISFoundation/dockerfiles/tree/master/pip-kit)

in requirements.in you should add the constraint ``jinja2>=2.10.1``

Comment (pcrespov) [reply requirements.txt]: Try using pip-kit and if it is not easy to use or unclear, please let me know. By using the tools we get them fully tuned to our needs :-)

Comment (sanderegg) [reply requirements.txt]: hmm which is what I did, see [requirements.in](requirements.in)

Comment (pcrespov) [reply requirements.txt]: OK. My bad I did not see the comment  after jinja entry  i.e. ``# via cookiecutter, jinja2-time`` and thought it was modified by hand

**Pull request merged**